cmd/dashboards/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/grafana/grafana-foundation-sdk/go/dashboard"
+
+	"github.com/grafana/cloudcost-exporter/cloudcost-exporter-dashboards/dashboards"
+)
+
+func main() {
+	output := flag.String("output", "console", "Where to write output to. Can be console or file")
+	outputDir := flag.String("output-dir", "./cloudcost-exporter-dashboards/grafana", "output directory")
+	flag.Parse()
+	dashes := dashboards.BuildDashboards()
+
+	err := run(dashes, output, outputDir)
+	if err != nil {
+		log.Fatalf("error generating dashboards: %s", err.Error())
+	}
+}
+
+func run(dashes []*dashboard.DashboardBuilder, output *string, outputDir *string) error {
+	for _, dash := range dashes {
+		build, err := dash.Build()
+		if err != nil {
+			return err
+		}
+		data, err := json.MarshalIndent(build, "", "  ")
+		if err != nil {
+			return err
+		}
+		if *output == "console" {
+			fmt.Println(string(data))
+			continue
+		}
+		err = createFile(*outputDir, sluggify(*build.Title), data)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// sluggify will take a string and convert the string
+func sluggify(s string) string {
+	s = strings.TrimSpace(s)
+	return strings.ReplaceAll(strings.ToLower(s), " ", "-")
+}
+
+func createFile(outputDir, name string, data []byte) error {
```

Comment:
Nitpick: https://pkg.go.dev/os#WriteFile could probably be used instead :)

Reply:
Nice! I always forget the _many_ ways of creating a file in Go. Let me refactor it to use that since it's simpler